qprofiler2/test/org/qcmg/qprofiler2/summarise/ReadGroupSummaryTest.java

```diff
@@ -0,0 +1,394 @@
+package org.qcmg.qprofiler2.summarise;
+ 
+import static org.junit.Assert.assertTrue;
+import org.junit.After;
+import org.junit.Test;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.qcmg.picard.SAMFileReaderFactory;
+import org.qcmg.qprofiler2.bam.BamSummarizer2;
+import org.qcmg.qprofiler2.bam.BamSummaryReport2;
+import org.qcmg.qprofiler2.summarise.ReadGroupSummary;
+import org.qcmg.qprofiler2.util.XmlUtils;
+import org.w3c.dom.Element;
+
+import org.qcmg.common.string.StringUtils;
+import org.qcmg.common.util.XmlElementUtils;
+
+import htsjdk.samtools.SAMRecord;
+import htsjdk.samtools.SamReader;
+
+public class ReadGroupSummaryTest {
+	protected static final String INPUT_FILE = "input.sam";
```

Comment:
could you pls use junits TemporaryFolder when creating files in tests

Reply:
All tmp file inside junit test moved into TemporaryFolder